api/WebSocket.json

```diff
@@ -23,28 +23,17 @@
               "version_removed": "11",
               "prefix": "Moz",
               "notes": "Message size limited to 16 MB (see <a href='https://bugzil.la/711205'>bug 711205</a>)."
-            },
-            {
-              "version_added": "4",
-              "version_removed": "6",
-              "notes": "Message size limited to 16 MB (see <a href='https://bugzil.la/711205'>bug 711205</a>)."
             }
           ],
           "firefox_android": [
             {
-              "version_added": "14",
-              "notes": "See <a href='https://bugzil.la/695635'>bug 695635</a>."
+              "version_added": "14"
```

Comment:
Can you check where this version 14 (as opposed to 11 for desktop) cam from? I might have been different for desktop and Android, but it seems just as likely to be a mistake. Since I'm asking you to copy the data to the constructor, it would be good to check it first.